utensor_cgen/code_generator.py

```diff
@@ -41,15 +48,19 @@ def generate(self, src_fname):
 
     opFactory = OperatorFactory()
 
-    print("Parsing {}".format(self.pb_file))
-    ops_info, ops_torder, output_nodes = parse_pb(self.pb_file, self.output_nodes)
-    construct_order = Optimizer.optimize(ops_info, ops_torder, output_nodes, self.method)
+    print("Transforming graph: {}".format(self.pb_file))
+    new_pb_file = self._transform_graph(self.pb_file)
+    print("Parsing new pb file {}".format(new_pb_file))
+    ops_info, topo_orders = parse_pb(new_pb_file, self.output_nodes)
```

Comment:
No. It's really a new pb file.
I use a temp file to store the new graph.
Since I don't want to refactor `parse_pb` for now.
It was designed to parse a pb file before.
It can be refactored to parse a graph def but it's just some extra work I don't want to deal with for now.